src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -252,30 +252,6 @@ public async Task RazorViewEngine_RendersPartialViews(string actionName, string
             Assert.Equal(expected, body.Trim(), ignoreLineEndingDifferences: true);
         }
 
-        [Fact(Skip = "https://github.com/aspnet/Mvc/issues/8754")]
-        public Task RazorViewEngine_RendersViewsFromEmbeddedFileProvider_WhenLookedupByName()
-            => RazorViewEngine_RendersIndexViewsFromEmbeddedFileProvider("/EmbeddedViews/LookupByName");
-
-        [Fact(Skip = "https://github.com/aspnet/Mvc/issues/8754")]
-        public Task RazorViewEngine_RendersViewsFromEmbeddedFileProvider_WhenLookedupByPath()
-            => RazorViewEngine_RendersIndexViewsFromEmbeddedFileProvider("/EmbeddedViews/LookupByPath");
-
-        private async Task RazorViewEngine_RendersIndexViewsFromEmbeddedFileProvider(string requestPath)
```

Comment:
I see you removed the web site's support for this. Would have thought embedded views would work if runtime complication is enabled. Is the combination tested elsewhere?

Reply:
The functional tests use an in-memory file provider which acts pretty much like the embedded one. These specific tests were used to verify path manipulation for files not coming from the file provider which is covered by default due to Razor Sdk compiled views.